packages/common-all/src/types/index.ts

```diff
@@ -70,6 +70,10 @@ export type DendronSiteFM = {
   canonicalUrl?: string;
   nav_order?: number;
   nav_exclude?: boolean;
+  /**
+   * Should exclude children from nav
+   */
+  nav_children_exclude?: boolean;
```

Comment:
should we name this as `nav_exclude_children` instead?

Reply:
Also how about `nav_include_children`, where the default is true? If I remember it right we were trying to go with positive names for configs (e.g. enableX instead of disableX), it might make sense to follow that here too.

Reply:
we switched to `enable*` in the `dendron.yml`. for this case, since the overwhelming case is that children should be included and because `nav_exclude` is also still a property, would follow the "stick to existing conventions" part for this prop